src/fragmentarium/ui/display/Transliteration.tsx

```diff
@@ -143,16 +150,48 @@ function DisplayLine({
   )
 }
 
+function DisplayRulingDollarLine({
+  line,
+  container = 'div'
+}: {
+  line: Line
+  container?: string
+}): JSX.Element {
+  const rulingDollarLine = line as RulingDollarLine
+  const element = `Transliteration__${rulingDollarLine.type}`
+  const number = capitalizeFirstLetter(
+    rulingDollarLine.display_value.split(' ')[0]
+  )
+  return React.createElement(
+    container,
+    { className: classNames([element]) },
+    <div className={`${element}${number}`} />
```

Comment:
Use `hr` for rulings. 

Having two nested dollar line elements seems weird. What about using a modifiuer instead. E.g. `Transliteration__RulingDollarLine Transliteration__RulingDollarLine--SINGLE`.

Reply:
So i used `.SINGLE`instead of modifiers. 
I wasn't able to create the line effect with `<hr/>' . Using `<hr/>`with *border-top & border-top* leads to the middle horizontal line (`<hr/>') not to be centered in the screen. I could alternatively use three `<hr/> within one `<div>`thats scaled.

Reply:
Why do you invent your conventions instead of using BEM?

Why do you need a `div` with `hr`?

What's the issue with horizontal centring? There should no difference between `hr` and `div. You know all the sizes so it should be a matter of adding correct margins.

Reply:
If you do not like to use modifiers, you could have elements `Transliteration__single-ruling` etc.

Reply:
That might be semantically more "correct" than letting ATF terminology leak into CSS.

Reply:
Ok to use `Transliteration__single-ruling` i would have to manipulate the `displayValue`property by replacing the space with and - .
And for the `<hr>`tag.  I dont think its possible to create three horizontal lines with borders within `<hr>`.

Reply:
Do not use `displayValue`.

Reply:
Why not? It is a HTML-element like any other. It just has some default styles.

Reply:
i could use a map
```javascript
const rulingNumber = new Map[
["SINGLE", "single-ruling"],
["DOUBLE", "single-ruling"],
["TRIPLE", "single-ruling"]
]
```
to create the css classnames

Reply:
we will need `displayValue` for stateLine though right ?

Reply:
The same styles which work for the `div` should work for the `hr`.

Reply:
ok. I will try it some more. And to create the css classnames i'll use a map and match the `number` property of RUlingDollarLine against this map ?

Reply:
You can use the map or convert `number` to lower case.

We do not need `displayValue` at all. All the required data is already there, but it might be easier to create that in the API, since we already have the logic for `content`. 

Display value makes no sense for the rulings since they are not displayed as text.

Reply:
ok <hr> is now implemented